src/Dynamo/src/Dynamo.CSLang/CSUsing.cs

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Isn't this part of the emitter's environment? This should be known by parsing the ABI.